contrib/sudo-pair.spec

```diff
@@ -0,0 +1,40 @@
+Name: sudo-pair
+Version: v0.9.2
+Release: 1
+Summary: Plugin for sudo that requires another human to approve and monitor privileged sudo sessions.
+Group: tem Environment/Libraries
+License: Apache Software License 2.0
+Url: https://github.com/square/sudo_pair
+Source: https://github.com/square/sudo_pair/archive/sudo_pair-%{version}.tar.gz
+
+BuildRoot: %{_tmppath}/%{name}-%{version}-build
+BuildRequires: cargo
+BuildRequires: clang-devel
+BuildRequires: git
+Requires: sudo
+
+%description
+Plugin for sudo that requires another human to approve and monitor privileged sudo sessions
+
+%global debug_package %{nil}
+
+%prep
+
+%setup -n sudo_pair-sudo_pair-%{version}
+
+%build
+cargo build --release
+
+%install
+mkdir -p %{buildroot}/usr/libexec/sudo
+%{__cp} target/release/libsudo_pair.so %{buildroot}/usr/libexec/sudo/
```

Comment:
I'd also suggest putting the prompts /etc, the approval script into /usr/bin, and having sample `/etc/sudo.conf` configuration in `/usr/share/doc/sudo_pair`.

Reply:
I'll work on that.

Reply:
👍